pkg/rpc/request/param_test.go

```diff
@@ -19,9 +20,13 @@ func TestParam_UnmarshalJSON(t *testing.T) {
                  {"cosigner": "f84d6a337fbc3d3a201d41da99e86b479e7a2554"},
                  {"sender": "f84d6a337fbc3d3a201d41da99e86b479e7a2554", "cosigner": "f84d6a337fbc3d3a201d41da99e86b479e7a2554"},
                  {"contract": "f84d6a337fbc3d3a201d41da99e86b479e7a2554"},
-                 {"state": "HALT"}]`
+                 {"state": "HALT"},
+                 {"account": "0xcadb3dc2faa3ef14a13b619c9a43124755aa2569", "scopes": 0},
+                 [{"account": "0xcadb3dc2faa3ef14a13b619c9a43124755aa2569", "scopes": 0}]]`
```

Comment:
BTW, have you cross-checked `scopes` JSON serialization for `Cosigner` with C# node? Last time I've checked it seemed to be using numbers, but at the same time there is [an example in RPC documentation](https://docs.neo.org/v3/docs/en-us/reference/rpc/latest-version/api/getrawtransaction.html) that contains `"scopes": "CalledByEntry"`, so I'm wondering what's going on here.

Reply:
They use flag enums for marshalling [scope](https://github.com/neo-project/neo/blob/master/src/neo/Network/P2P/Payloads/Cosigner.cs#L54). As far as I know, the default behavior for them is to be serialised as numbers (not as strings from names), also they don't have any serializing attributes over [WitnessScope](https://github.com/neo-project/neo/blob/master/src/neo/Network/P2P/Payloads/WitnessScope.cs#L5), so it's definitely numbers. Perhaps, they should update documentation.

Reply:
And in case when we use strings for scope, we should be able to combine them together (e.g. `scopes` should be an array of strings). However, example in RPC documentation assumes there could be only one scope.